dags/utils/dag_eo_utils.py

```diff
@@ -483,8 +489,8 @@ def create_actors(**kwargs):
                 df["statut"] = df["public_accueilli"].apply(
                     lambda x: "SUPPRIME" if x == "Professionnels" else "ACTIF"
                 )
-            elif old_col == "uniquement_sur_rdv":
-                df[new_col] = df[old_col].astype(bool)
+            elif old_col in ["uniquement_sur_rdv", "exclusivite_de_reprisereparation"]:
+                df[new_col] = _column_to_bool(df[old_col])
```

Comment:
Je suis partagé sur le fait de sortir le apply dans la fonction, je serais tenté d'écrire ca comme 
```suggestion
                df[new_col] = df[old_col].apply(cast_eo_boolean_string_to_boolean)
```

Et du coup 

```py
def cast_eo_boolean_string_to_boolean(value: str | bool) -> str:
  if isinstance(value, bool):
    return row_value 
  if isinstance(value, str):
    return value.lower().strip() == "oui":
```

Reply:
c'est un poil plus verbeux mais je trouve plus lisible, et conserve une écriture panda-esque

Reply:
bon le nom de fonction est volontairement long...on peut faire bien mieux

Reply:
👍